website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -51,19 +60,21 @@ The green checkmark means the dbt builds and tests were successful. Clicking the
 The green checkmark means the dbt builds and tests were successful. Clicking on the dbt Cloud section navigates you to the relevant CI run in dbt Cloud.
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Enabling-CI/ADO CI Check.png" title="Azure DevOps pull request"/>
 
-## Configuring a dbt Cloud CI job
+## Configuring a Slim CI job
 
-Setting up a CI job is very similiar to setting up a normal production job that runs on a schedule; however, a CI job has some notable differences.
+Slim CI offers an alternative in dbt Cloud for running and testing all models in your project. When you're only changing a few handful of models, Slim CI can decrease the time it takes by running and testing only modified models. Slim CI enables you to focus on testing changed models and reduces your overall cost. 
 
-There are a few components that define a Slim CI job.
+Once you open a pull request, dbt Cloud builds the models affected by the code change in a temporary schema and runs the tests for these models as a check. This process provides a staging environment where you can check the run status. When the CI job completes, you can see the run status directly in the pull request. The CI job enables you to deploy new code to production with confidence.
```

Comment:
This statement is true whether or not Slim CI is enabled, and is very similar to the stuff on line 27. I'd move this up and combine the best bits of both